src/apps/components/AppDetailsPage/PermissionsCard.tsx

```diff
@@ -12,22 +13,42 @@ type PermissionsCardProps = {
     code: PermissionEnum;
   }> | null;
   loading: boolean;
+  appId: string; // todo wrap with App Context
 } & BoxProps;
 
 export const PermissionsCard: React.FC<PermissionsCardProps> = ({
   permissions,
   loading,
+  appId,
   ...boxProps
 }) => {
+  const [editPermissionDialogOpen, setEditPermissionDialogOpen] =
+    useState(false);
   const intl = useIntl();
 
+  const editPermissionsButton = (
+    <Button
+      variant={"secondary"}
```

Comment:
**question**: do you prefer to use brackets also for strings?

Reply:
Its a funny thing. I don't. But I use Jetbrains IDE and it has a bug with Macaw - it doesn't give me autocomplete when I use `=""`, but with `={""}` it does. I opened an issue for them - this is a problem with their IDE - they have different mechanisms for these syntaxes.

Since this is a preference of everyone AND it can be handled and fixed by ESlint, I suggest to add a rule for that.
Then, I can write with `{}` and after I commit, everything will be transformed to `""`

https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-curly-brace-presence.md